src/generate/rust.rs

```diff
@@ -143,6 +144,11 @@ impl<Pat: Ord + Hash + RustInputPat> Rc<Rule<Pat>> {
                 elem.parse_node_kind(parse_nodes).0,
                 sep.parse_node_kind(parse_nodes).0
             )),
+            Rule::RepeatManyTrailing(elem, sep) => ParseNodeKind(format!(
+                "({}* %% {})",
+                elem.parse_node_kind(parse_nodes).0,
+                sep.parse_node_kind(parse_nodes).0
+            )),
             Rule::RepeatMore(rule, None) => ParseNodeKind(format!(
                 // FIXME(rust-lang-nursery/rustfmt#3004) work around rustfmt removing trailing +.
```

Comment:
(issue seems closed)

Reply:
Please open an issue on lykenware/gll for cleanup.